services/auth.service.js

```diff
@@ -0,0 +1,68 @@
+const authUtil = require("../utils/auth.util");
+const validationUtil = require("../utils/validation.util");
+const userService = require("../services/user.service");
+require("dotenv").config();
+
+const registerUser = async (body) => {
+  const response = await userService.createUser(body);
+  if (response.status==201) {
+ 
+
+    const token = await authUtil.generateJwtToken(newUser.username);
+    
+    return {status: 201, message: token};
+  } else {
```

Comment:
this else is not needed. As the if condition has a return function.
```suggestion
    }
    return {status: response.status, message: response.message};
```

Reply:
I have deleted this.